packages/block-editor/src/components/inserter/block-types-tab.js

```diff
@@ -137,13 +137,13 @@ export function BlockTypesTab( {
 				{ didRenderAllCategories && uncategorizedItems.length > 0 && (
 					<InserterPanel
 						className="block-editor-inserter__uncategorized-blocks-panel"
-						title={ __( 'Uncategorized' ) }
+						title={ _x( 'Uncategorized', 'block category' ) }
```

Comment:
In this case (and many others), context was added because the translation may vary depending on the context (masculine/feminine/neutral etc)